Data/Global.lua

```diff
@@ -103,10 +103,11 @@ KeywordFlag.Attack =	0x00010000
 KeywordFlag.Spell =		0x00020000
 KeywordFlag.Hit =		0x00040000
 KeywordFlag.Ailment =	0x00080000
+KeywordFlag.Brand =		0x00100000
 -- Other effects
-KeywordFlag.Poison =	0x00100000
-KeywordFlag.Bleed =		0x00200000
-KeywordFlag.Ignite =	0x00400000
+KeywordFlag.Poison =	0x00200000
+KeywordFlag.Bleed =		0x00400000
```

Comment:
When I changed just these 3 lines locally, this caused tree nodes for bleeding to start reporting as Poison increases

Reply:
You have to refresh the ModCache by pressing ctrl + f5 then it will work properly